.github/workflows/go.yml

```diff
@@ -20,26 +24,30 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.22.1'
+          go-version-file: ./go.mod
 
       - name: Build binary on Linux and macOS
         if: matrix.os != 'windows-latest'
+        env:
+          OS: ${{ matrix.os == 'ubuntu-latest' && 'linux' || 'darwin' }}
```

Comment:
Rebrand the `ubuntu-latest` binary to `linux`, and the `macos-latest` binary to `darwin`.

Reply:
strongly suggest removing the "put inside dmg" step on macos/darwin while you're at it

Reply:
As I'm not on the project, or involved in any way, I don't want to remove any functionality in this PR.  That's a decision for the maintainers.

I have, however, updated the pipeline to keep the MacOS binary around as well as the dmg in [3b35d88](https://github.com/danielmiessler/fabric/pull/810/commits/3b35d88611d8bfae01476c0e2a833db1e2300d0e).